File: src/main/scala/com/scala/chapter01/Student.scala
package com.scala.chapter01

class Student(name: String, var age: Int) {

  def printInfo(): Unit = {
    println(name + " " + age + " " + Student.school)

  }
}

object Student {
  val school: String = "scala"

  def main(args: Array[String]): Unit = {
    val stu = new Student("sanshan", 20)
    stu.printInfo()
  }
}

File: src/main/java/com/scala/entity/Student.java
package com.scala.entity;

public class Student {

    private String name;
    private Integer age;
    private static String school = "scala";

    public Student(String name, Integer age) {
        this.name = name;
        this.age = age;
    }

    public void printInfo() {
        System.out.println(this.name + " " + this.age + " " + Student.school);
    }


    public static void main(String[] args) {
        Student stu = new Student("cang", 19);
        stu.printInfo();


    }
}
